<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>WBS</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_result">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>369</y>
      <width>941</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255);
border:1px solid #aaa;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget_result">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>941</width>
       <height>261</height>
      </rect>
     </property>
     <property name="autoScrollMargin">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_log">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>940</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255);
border:1px solid #aaa;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>921</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_scanresult">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>扫描结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_loginfo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日志信息：</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_option" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>940</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #aaa;</string>
    </property>
    <widget class="QLabel" name="label_target">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#000;border:0px;</string>
     </property>
     <property name="text">
      <string>目标：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_target">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid #aaa;</string>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_end">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid #aaa;</string>
     </property>
     <property name="text">
      <string>结束</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#000;border:0px;</string>
     </property>
     <property name="text">
      <string>扫描速度：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_speed">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>低</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>高</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_php">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>PHP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_asp">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>ASP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_jsp">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>JSP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_aspx">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>120</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>ASPX</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_others">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>90</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>Others</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_user_defined">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>120</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>User-Defined</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_php_num">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_asp_num">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_aspx_num">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jsp_num">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_others_num">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_user_defined_num">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_delay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#000;border:0px;</string>
     </property>
     <property name="text">
      <string>设置延迟：</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_delay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>mm</string>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_status_code_200">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>200</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_status_code">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#000;border:0px;</string>
     </property>
     <property name="text">
      <string>状态码：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_status_code_403">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>403</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_status_code_3XX">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>3XX</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_status_code_500">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>500</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_suffix">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#000;border:0px;</string>
     </property>
     <property name="text">
      <string>后缀：</string>
     </property>
    </widget>
    <widget class="Line" name="line_left">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>54</y>
       <width>1</width>
       <height>85</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_right">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>54</y>
       <width>1</width>
       <height>85</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid #aaa;</string>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_detail">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>197</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>显示细节</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menufile1">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_setting"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menufile1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_setting">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>配置</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
